app/controllers/hold_controller.rb

```diff
@@ -0,0 +1,23 @@
+class HoldController < ApplicationController
+  before_action :require_user
+  before_action :authenticate_user!
+  load_and_authorize_resource
+  protect_from_forgery with: :exception
+
+  def show
```

Comment:
`render layout: false` would allow you to keep this out of our public UI template. (noting this as you mentioned it in the PR info, not because I specifically feel like we should do it).

Reply:
Thanks, that's good to know. I think I'll wait to change anything until Mikki reviews it (largely due to laziness, since using the layout gives me all the CSS I need).